### ISSUE TITLE
ObjC method arrayWithObjects: requires 1 argument, but JavaScript passed 4 arguments

### ISSUE THREAD

**Issue by mackelito** [mathieudutour/git-sketch-plugin#155]
### How did it happen?
1. Changed a file
2. Tried to commit

### Error log

```
ObjC method arrayWithObjects: requires 1 argument, but JavaScript passed 4 arguments
```

**Comment by mackelito** [comment]
Perhaps this is something related to High Sierra?

**Comment by mackelito** [comment]
I guess there has been changes to Sketch api when using NSArray..

**Comment by yannicgraeser** [comment]
I have the same problem.
And when I deinstall the plugin I can't reinstall it. It is not showing up in the plugins menu :(

**Comment by azzurrodomoto** [comment]
I have the same problem, too.
Advanced > Init git repo, then I saw the alert window.

**Comment by mathieudutour** [comment]
fixed in v0.11.3, thanks @notandrewkaye!
